packages/fluentui/react-icons-northstar/src/components/BreakoutRoomIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const BreakoutRoomIcon = createSvgIcon({
+  svg: ({ classes }) => (
+    <svg viewBox="0 0 12 16" fill="none" className={classes.svg} role="presentation">
+      <path
+        className={cx(iconClassNames.filled, classes.filledPart)}
```

Comment:
I don't think that this is accurate... Any suggestions?

![image](https://user-images.githubusercontent.com/14183168/87543412-ff658180-c6a4-11ea-98fe-1156a1b687bc.png)

Reply:
Updated